src/components/MessageBanner.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+
+export const MessageBanner = ({
+  message = "There's no movie added to My List yet",
```

Comment:
As this is a message to the user it should be added to the language file, I think it is called en.js